models/curvefi/ethereum/curvefi_ethereum_view_pools.sql

```diff
@@ -106,10 +93,10 @@ plain_pools_deployed AS (
         _fee AS out_fee,
         output_0 AS token_address,
         output_0 AS deposit_contract,
-        _coins [1] AS coin0,
-        _coins [2] AS coin1,
-        _coins [3] AS coin2,
-        _coins [4] AS coin3,
+        _coins [0] AS coin0,
```

Comment:
great catch -- i know it's in the docs, but i wonder how we best educate the array index differences. may not be much of an issue post-migration days of copying old scripts

Reply:
yeah I think it's just a migration issue